src/drafts/SelectPanel2/SelectPanel.tsx

```diff
@@ -127,7 +135,9 @@ const SelectPanel = props => {
                   container: 'SelectPanel',
                   listRole: 'listbox',
                   selectionAttribute: 'aria-selected',
-                  selectionVariant: props.selectionVariant || 'multiple',
+                  selectionVariant:
```

Comment:
Can we do 

```jsx
selectionVariant: props.selectionVariant === 'instant' ? 'single' : props.selectionVariant,
```
since we have `multiple` as the default value specified above? 🤔

Reply:
Good tip! We definitely can

```suggestion
                  selectionVariant: props.selectionVariant === 'instant' ? 'single' : props.selectionVariant,
```